### PR TITLE
Use Tox to build out RST files

### PR DESCRIPTION
**Pull request by sc68cal**
This commit adds support for Tox, and creates a directory structure
similar to the docs of other projects, where the RST files are located
in doc/source and the artifacts are placed in doc/build.

Tox is useful because it will track the python & sphinx dependencies and
also is the toolchain of choice for other repositories that use RST
files for docs, such as the specs repositories.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -0,0 +1,23 @@
+[metadata]
+name = OpenStack First Application
+summary = OpenStack Fist Application
```
Comment (berendt) [inline setup.cfg]: Type. Should be "First".

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+description-file =
+    README.rst
+author = OpenStack
```
Comment (berendt) [inline setup.cfg]: I would prefer "OpenStack Documentation" here.

File: tox.ini [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+install_command = pip install {opts} {packages}
+
+[testenv:docs]
+deps =
+   Pygments
+   docutils
+   sphinx>=1.1.2,<1.2
+   pbr>=0.6,!=0.7,<1.0
+   oslosphinx
+   openstackdocstheme
```
Comment (berendt) [inline tox.ini]: I would prefer to use an external `test-requiremens.txt` file here.

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+# Copyright (c) 2013 Hewlett-Packard Development Company, L.P.
```
Comment (berendt) [inline setup.py]: Can we drop the copyright statement?

File: setup.cfg [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+author-email = openstack-docs@lists.openstack.org
+home-page = docs.openstack.org
```
Comment (berendt) [inline setup.cfg]: http://doc.openstack.org/

**Comment by sc68cal** [comment]
@berendt updated

**Comment by berendt** [comment]
Tested. Works for me.

**Pull request merged**